cfn-guard-rulegen-lambda/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "cfn-guard-rulegen-lambda"
+version = "0.1.0"
+authors = ["John Capps <jocapp@amazon.com>"]
+edition = "2018"
+
+[dependencies]
```

Comment:
Can you double-check that these are the same versions as what's in cfn-guard-lambda?